test/logquantuniform.jl

```diff
@@ -2,36 +2,38 @@
 Based on
 https://hyperopt.github.io/hyperopt/
 """
-module TestQuantLogUniform
+module TestLogQuantUniform
 
 using Test
 using TreeParzen
+#using Distribution
 
 @testset "quantised log uniform" begin
 
-    qlu = HP.QuantLogUniform(:qlu, log(1.01), log(20.0), 2.0)
+    qlu = HP.LogQuantUniform(:qlu, log(1.0), log(20.0), 2.0)
+
     N = 10_000
 
     qlu_samples = [TreeParzen.Resolve.node(qlu, TreeParzen.Trials.ValsDict()) for i in 1:N]
+    @test 1.0 <= minimum(qlu_samples)
+    @test maximum(qlu_samples) <= 20.0
```

Comment:
More tests needed. You need to relate the size of the gap between the `log` of the unique samples to the value of `q` given on line 13.